stravalib/model.py

```diff
@@ -926,10 +1241,35 @@ class SubscriptionCallback(
     hub_verify_token: Optional[str] = None
     hub_challenge: Optional[str] = None
 
+    # TODO: error: Required dynamic aliases disallowed  [pydantic-alias]
     class Config:
         alias_generator = lambda field_name: field_name.replace("hub_", "hub.")
 
```

Comment:
this is another override type issue. i think once i understand overrides i can easily fix them all.